client/python/samples/README.md

```diff
@@ -0,0 +1,223 @@
+# OpenVINO&trade; Model Server Client Library Example Clients
+
+This document contains examples to run *GetModelStatus*, *GetModelMetadata*, *Predict* functions over gRPC API.
+
+It covers following topics:
+* <a href="#grpc-api">gRPC API Example Clients </a>
+
+## Requirement
+
+**Note**: Provided examples and their dependencies are updated and validated for Python 3.6+ version. For older versions of Python, dependencies versions adjustment might be required.
```

Comment:
Python 3.6+ is a requirement for the library itself to work. I think we can drop this part.